.github/workflows/_rust.yml

```diff
@@ -129,9 +129,7 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - uses: ./.github/actions/setup-rust
-      - run: cargo build -p firezone-headless-client
-        name: "cargo build"
-        shell: bash
+      - uses: ./.github/actions/setup-tauri
```

Comment:
Because of how Cargo works, the IPC service needs the GUI dependencies to be installed, even though they don't end up compiled into the binary.

Reply:
👍